velox/substrait/tests/JsonToProtoConverter.h

```diff
@@ -24,4 +24,8 @@ class JsonToProtoConverter {
   static void readFromFile(
       const std::string& msgPath,
       google::protobuf::Message& msg);
+
+  /// Convert Protobuf message to Json String.
```

Comment:
JSON is a string format, hence, saying Json String is redundant. Replace Json String with JSON.